specification/storage/resource-manager/readme.azureresourceschema.md

```diff
@@ -6,174 +6,154 @@ These settings apply only when `--azureresourceschema` is specified on the comma
 
 ``` yaml $(azureresourceschema) && $(multiapi)
 batch:
-  - tag: schema-storage-2019-06-01
-  - tag: schema-storage-2019-04-01
-  - tag: schema-storage-2018-11-01
-  - tag: schema-storage-2018-07-01
-  - tag: schema-storage-2018-03-01-preview
-  - tag: schema-storage-2018-02-01
-  - tag: schema-storage-2017-10-01
-  - tag: schema-storage-2017-06-01
-  - tag: schema-storage-2016-12-01
-  - tag: schema-storage-2016-05-01
-  - tag: schema-storage-2016-01-01
-  - tag: schema-storage-2015-06-15
-  - tag: schema-storage-2015-05-01-preview
-
+  - tag: package-2020-08-preview
+  - tag: package-2019-06
+  - tag: package-2019-04
+  - tag: package-2018-11
+  - tag: package-2018-07
+  - tag: package-2018-07-only
+  - tag: package-2018-03
+  - tag: package-2018-02
+  - tag: package-2017-10
+  - tag: package-2017-06
+  - tag: package-2016-12
+  - tag: package-2016-05
+  - tag: package-2016-01
+  - tag: package-2015-06
+  - tag: package-2015-05-preview
```

Comment:
@leni-msft to review as this file is not need change.

Reply:
@huizlAzure Please run `npm run multiapi` to update this readme.azureresourceschema.md file. (may need sync your work branch with master first)

Reply:
@leni-msft I verified running npm run multiapi and it didn't changes any file locally.

> azure-rest-api-specs-tests@0.1.0 multiapi E:\Code\Swagger\James-Repo\azure-rest-api-specs
> ts-node ./scripts/multiapi.ts

Why do you think this file need not change? I see all the api-versions are listed in this file and this new api-version needs to be added as well.

Reply:
Hi @priyapansrp, this file do need to be updated, by adding new `schema-storage-{APIVersion}` tag, but not using the `package-*` tags. I pushed the commit https://github.com/Azure/azure-rest-api-specs/pull/10570/commits/fd8466803ac15c63995b9b6f9b088f13debfae1a to this PR. Let's see the `automation - sdk (SDK Schema) ` result.